gradle/libs.versions.toml

```diff
@@ -13,7 +14,7 @@ errorproneGradle = "3.0.1"
 intellij-platform = "2.0.1"
 jdk = "22"
 jvmTarget = "17"
-jewel = "0.15.2.2"
+jewel = "0.23.0"
```

Comment:
Let's use the latest version

Reply:
eae65724045d598a94a4c65b6fa00ea99ef68d22

Because of these last two commits, the painter resource for my Icon was deprecated, so I need to work on fixing that:
'painterResource(String): Painter' is deprecated. Migrate to the Compose resources library. See https:// www. jetbrains. com/ help/ kotlin-multiplatform-dev/ compose-images-resources. html If you need to load resources specificly from Java classpath, you should still use the new resource library and use a snippet from https:// github. com/ JetBrains/ compose-multiplatform-core/ pull/ 1457

Reply:
I paired with Linh and updated it and now it works. I used the suggested migration snippet they posted https://github.com/JetBrains/compose-multiplatform-core/pull/1457 for the painter resource. I will keep an eye out in case there are any changes.